### ISSUE TITLE
Event properties

### ISSUE THREAD

**Issue by pewallin** [danscan/rn-redux-mixpanel#2]
Are you in the process of adding them? Noticed that trackEvent takes a eventData object. Maybe add it to the middlware signature as "selectProperties = ()=>null"? and

```
const distinctId = selectDistinctId(action, state)
const properties = selectProperties(action, state)

// Track action event with Mixpanel
trackEvent({
  token,
  distinctId,
  eventName: action.type,
  eventData: properties
})
```

**Comment by danscan** [comment]
Really good idea @pewallin .  I hadn't added it yet, since the properties I was initially trying to send were overflowing the URL char limit, even after base64 encoding (as Mixpanel's HTTP API requires).

**Comment by danscan** [comment]
If you want to create a PR, I'll gladly accept; otherwise I can get to it tomorrow.

**Comment by danscan** [comment]
Forgot to close this when we merged #3
